spring-cloud-data-admin/src/main/java/org/springframework/cloud/data/admin/config/CloudConfiguration.java

```diff
@@ -14,21 +14,23 @@
  * limitations under the License.
  */
 
-package org.springframework.cloud.data.rest.config;
+package org.springframework.cloud.data.admin.config;
 
 import org.springframework.cloud.data.module.deployer.ModuleDeployer;
 import org.springframework.cloud.data.module.deployer.cloudfoundry.CloudFoundryModuleDeployer;
 import org.springframework.cloud.data.module.deployer.lattice.ReceptorModuleDeployer;
 import org.springframework.context.annotation.Bean;
+import org.springframework.context.annotation.Configuration;
 import org.springframework.context.annotation.Profile;
 
 /**
- * Configuration for cloud profiles (cloud, lattice)
+ * Configuration for cloud profiles (cloud, lattice).
  *
  * @author Mark Fisher
  * @author Thomas Risberg
  */
 @Profile("cloud")
+@Configuration
```

Comment:
minor thing but should we have `@Profile` on the line after `@Configuration` since this class is primarily Configuration but has its applicability narrowed by the Profile specified? Also, I believe that would be consistent with other cases within the project.